nativescript-angular/router/ns-router-link.ts

```diff
@@ -30,30 +30,41 @@ import {routerLog} from "../trace";
  */
 @Directive({ selector: '[nsRouterLink]' })
 export class NSRouterLink {
-  private commands: any[] = [];
-  @Input() target: string;
-  @Input() queryParams: { [k: string]: any };
-  @Input() fragment: string;
+    private commands: any[] = [];
+    @Input() target: string;
+    @Input() queryParams: { [k: string]: any };
+    @Input() fragment: string;
 
-  /**
-   * @internal
-   */
-  constructor(private router: Router, private route: ActivatedRoute) { }
+    urlTree: UrlTree;
```

Comment:
Needed by the nsRouterLinkActive directive while processing the reduceList function:

https://github.com/ntrp/nativescript-angular/blob/f78b8e874eb0aab3b42cd6c04c7abf44ec7f5768/nativescript-angular/router/ns-router-link-active.ts#L99